### PR TITLE
Add example for Prefix on ingress.md

### PR DESCRIPTION
**Pull request by dinilimento**
From my experience the example that was added is not matched.
If I understand correctly the path is checked on elements separated by a '/'. If the specified path prefixes the element that is checked it is a match. If there are multiple elements following the specified path prefix this will not result in a match.
 I found the documentation unclear on this. Adding this example might help.

### CODE REVIEW

**Review by Arhell — COMMENTED** [review]
Please sign CLA
https://github.com/kubernetes/website/pull/30648#issuecomment-979958008

File: content/en/docs/concepts/services-networking/ingress.md
Comment (sftim) [inline content/en/docs/concepts/services-networking/ingress.md:186]: I don't think this is correct. `/aaa/bbb` _is_ a prefix match for request path `/aaa/bbb/ccc/ddd`.

**Review by sftim — CHANGES_REQUESTED** [review]

**Comment by jimangel** [comment]
Love the intent of this PR, I would like to see tech confirmation from Tim's review before merging. Otherwise lgtm. Thanks!

/lgtm

File: content/en/docs/concepts/services-networking/ingress.md
Comment (reylejano) [reply content/en/docs/concepts/services-networking/ingress.md:186]: I believe if the request path is `/aaa/bbb/ccc/ddd` then **yes** it matches the prefix `/aaa/bbb` for the same reason why request path to `/aaa/bbb/ccc` matches with prefix `/aaa/bbb`

I'll do a test when I have cycles

**Comment by sftim** [comment]
/hold
to highlight that we want a tech review as well
/sig network

File: content/en/docs/concepts/services-networking/ingress.md
Comment (sftim) [reply content/en/docs/concepts/services-networking/ingress.md:186]: The actual behavior will depend on the ingress controller you select (watch out Rey in case you pick a noncompliant one)

Comment (dinilimento) [reply content/en/docs/concepts/services-networking/ingress.md:186]: For your reference: we use nginx-ingress in our setup. 
Sorry, I'm not able to add a sample at this time.
We ended up using the prefix but used a regex to let the path to match all that comes after the prefix.

File: content/en/docs/concepts/services-networking/ingress.md
Comment (kbhawkey) [inline content/en/docs/concepts/services-networking/ingress.md:185]: Note: There is an open question about prefix `/aaa/bbb` matching paths:
`/aaa/bbb/ccc` and `/aaa/bbb/ccc/ddd`

Ping @kubernetes/sig-network-misc

Comment (zacharysarah) [reply content/en/docs/concepts/services-networking/ingress.md:185]: @kubernetes/sig-network-pr-reviews 👋🏻

File: content/en/docs/concepts/services-networking/ingress.md
Comment (reylejano) [reply content/en/docs/concepts/services-networking/ingress.md:186]: Tested this with nginx-ingress
Requests to `/aaa/bbb/ccc/ddd` went to `/aaa/bbb`

**Comment by reylejano** [comment]
I tested the proposed changes in this PR with nginx as the ingress controller and did not get the same results what was proposed in this PR
From my tests, requests to `/aaa/bbb/ccc/ddd` went to `/aaa/bbb`

@dinilimento I'm closing this PR based on my test results
Feel free to reopen this PR with `/reopen` if your tests don't agree with my results

/close

**Pull request closed without merging**
